XenOvfTransport/Export.cs

```diff
@@ -354,6 +355,37 @@ private EnvelopeType _export(Session xenSession, string targetPath, string ovfna
                         OVF.AddOtherSystemSettingData(ovfEnv, vsId, "vgpu", sb.ToString(), OVF.GetContentMessage("OTHER_SYSTEM_SETTING_DESCRIPTION_4"));
                     }
                 }
+
+                var allProxies = PVS_proxy.get_all_records(xenSession).Select(kvp => kvp.Value).ToList();
+                
+                string pvsSiteUuid = string.Empty;
+
+                foreach (var p in allProxies.Where(p => p != null && p.VIF != null))
+                {
+                    var vif = xenSession.Connection.Resolve(p.VIF);
+                    if (vif != null)
+                    {
+                        var vmFromVif = xenSession.Connection.Resolve(vif.VM);
+                        if (vmFromVif != null && vmFromVif.uuid == vm.uuid)
+                        {
+                            var pvsSite = xenSession.Connection.Resolve(p.site);
+                            if (pvsSite != null)
+                            {
+                                pvsSiteUuid = pvsSite.uuid;
+                                
+                                break;

```

Comment:
I think that the `break `should be outside this `if `block, when we found the VM that this proxy is configured for. The site check is just to get site's uuid, not a condition for exiting the loop.